docs/source/index.rst

```diff
@@ -16,6 +16,7 @@ Mantid Documentation
    :glob:
    :maxdepth: 1
 
+   basics/index
```

Comment:
This renders as "Mantid Basics"

A few lines later you have "Mantid Basics Course" referring to the training course.  These titles are too similar for two separate but related things.

Reply:
Perhaps "Introduction to Mantid"?

Reply:
Also the index page itself is not a usefull index, it would be better to link to the Introduction page itself.